components/opponent/commons/opponent_display.lua

```diff
@@ -257,15 +296,17 @@ OpponentDisplay.propTypes.InlineTeam = {
 Displays a team as an inline element. The team is specified by a team struct.
 Only the default icon is supported.
 ]]
+---@param props {flip: boolean?, style: teamStyle?, team: standardTeamProps}
+---@return string
 function OpponentDisplay.InlineTeam(props)
 	DisplayUtil.assertPropTypes(props, OpponentDisplay.propTypes.InlineTeam)
-	return OpponentDisplay.InlineTeamContainer(Table.merge(props, {
+	return (OpponentDisplay.InlineTeamContainer(Table.merge(props, {
```

Comment:
Why's this needed?

Reply:
because it should only return the string and we have several gsubs being applied here so it would return 2 values